### PR TITLE
[WIP] frontend improvements

### PR DESCRIPTION
**Pull request by nemi-notrace**

### CODE REVIEW

**Comment by nemi-notrace** [comment]
I added `eslint-disable-next-line @typescript-eslint/no-redeclare` in player.tsx (line 38 & 47) to see if the build works @dryptomim 
Also I added ` eslint-disable-next-line no-useless-catch` in repo.ts (line 1047) @frando

**Comment by Frando** [comment]
LGTM. I'd be in favor of merging this and keep iterating on top. Ok?

**Comment by nemi-notrace** [comment]
> LGTM. I'd be in favor of merging this and keep iterating on top. Ok?

Yes! Do it.

File: packages/repco-frontend/app/components/primitives/input.tsx [hunk 1]
```diff
@@ -1,51 +1,113 @@
...
 import { cva, cx } from 'class-variance-authority'
 import type { ClassProp } from 'class-variance-authority/dist/types'
 
 const styles = cva(
-  'border transition-colors duration-100 cursor-default disabled:opacity-50',
+  'border rounded-lg transition-colors duration-100 outline-none focus:border-primary-500 disabled:opacity-50',
   {
     variants: {
       disabled: {
-        true: 'opacity-70 pointer-events-none cursor-not-allowed',
```
Comment (dryptomim) [inline packages/repco-frontend/app/components/primitives/input.tsx:10]: why remove "cursor-not-allowed"?  it is literally was "disabled" mean

Comment (nemi-notrace) [reply packages/repco-frontend/app/components/primitives/input.tsx:10]: Fixed it. I probably lost it when I rewrote the file...

**Pull request merged**